docker/istgt.Dockerfile

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 The OpenEBS Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:18.04 as build
+
+RUN mkdir -p istgt
+COPY . istgt/
```

Comment:
```suggestion
FROM ubuntu:18.04 as build
WORKDIR /istgt

COPY . .
```

Reply:
The code here depends on the libraries from ubuntu and as we have not yet tested the images properly with 20 we are using 18. Once we verify the 20 builds we can update this.